conjure-java-client-verifier/src/test/java/com/palantir/verification/Cases.java

```diff
@@ -31,23 +31,18 @@
 
 public final class Cases {
 
-    public static final ClientTestCases TEST_CASES = deserializeTestCases(
-            new File("build/test-cases/test-cases.json"));
-    private static final IgnoredClientTestCases IGNORED_TEST_CASES = deserializeIgnoredClientTestCases(
-            new File("src/test/resources/ignored-test-cases.yml"));
+    public static final ClientTestCases TEST_CASES = deserializeTestCases(new File("build/test-cases/test-cases.json"));
+    private static final IgnoredClientTestCases IGNORED_TEST_CASES =
+            deserializeIgnoredClientTestCases(new File("src/test/resources/ignored-test-cases.yml"));
 
     private Cases() {}
 
     private static ClientTestCases deserializeTestCases(File file) {
         try {
-            return new ObjectMapper()
-                    .registerModule(new Jdk8Module())
-                    .readValue(file, TestCases.class)
-                    .getClient();
+            return new ObjectMapper().registerModule(new Jdk8Module()).readValue(file, TestCases.class).getClient();
```

Comment:
This should have been fixed by 0.3.1 but I think it wasn't.. need to look into why tomorrow.

Reply:
https://github.com/palantir/palantir-java-format/pull/71/files#diff-f9b30f5f32648e37c5db4571679dd148